error.js

```diff
@@ -0,0 +1,11 @@
+const db = require('./db/connection')
+const commentError =(req,res,next)=>{
+    const reqBody = req.body
+    const { article_id } = req.params;
+    
+    if(!reqBody.username || !reqBody.body ){
```

Comment:
You don't need to manually check these - if they are missing they violate the seed which means psql will reject the query for you. See if you don't use this function whether you get an error in your catch block (console log it)